### ISSUE TITLE
Fix VisualScoreTagTest problem with travis CI

### ISSUE THREAD

**Issue by diogosantana** [psi-probe/psi-probe#618]
Fix VisualScoreTagTest problem as commented by @hazendaz at https://github.com/psi-probe/psi-probe/commit/f7ce9ac61117a57a4834260b8e362031077d610d.

````java
// TODO 1/18/16 This test was not previously used. It causes a lot of output
// and crashes travis CI.  Review it's need.
@Ignore
@Test
public void testRangeScan() {
````

**Comment by diogosantana** [comment]
@hazendaz Can you explain better why travis CI crashes?

Now that we using logback, I can make the output go away for test purporses. My it make me think if we should keep this info log on production code since we are now testing the code. By the way, the output is generated at https://github.com/psi-probe/psi-probe/blob/master/core/src/main/java/com/googlecode/psiprobe/jsp/VisualScoreTag.java#L121.

**Comment by hazendaz** [comment]
@diogosantana travis ci issue is related to output capped at 4mb.  They indicate that output of that size likely indicates an error in the test.  I didn't look much into this but that might be a possibility with the test itself.  Because surefire was manipulated from default settings, this project was not running this test case until now.  So it is quite possible the code is no longer valid with current library state.

**Comment by diogosantana** [comment]
I created the test to fix #466, so I know it is running since then. The output excess is due to VisualScoreTag class, not the test. It has four info log messages that may had a purpose in the past, but I believe it doesn't have one anymore, because now the class is tested.
I will remove the logs, reinstate the test and do a PR for this.

**Comment by hazendaz** [comment]
Sounds great!  Thanks.
